flight/PiOS/STM32/chibi_main.c

```diff
@@ -90,13 +92,62 @@ int main()
 
 	return 0;
 }
+
+#if defined(STM32F4XX)
+
+#include <stm32f4xx_flash.h>
+/**
+ * Check the brown out reset threshold is 2.7 volts and if not
+ * resets it.
+ */
+void check_bor()
```

Comment:
This really has nothing to do with ChibiOS. :/

Reply:
Neither does anything here, really-- it's the main() function and init task for stuff running under chibios.

Reply:
3 lines in main, and __early_init are the only chibi specific bits. P.S. https://github.com/mlyle/dronin/blob/9740da6184dde413a7e72c8dcedbe4fd63180edd/flight/PiOS/STM32/chibi_main.c#L6 is incorrect. 😉